server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const port = 3000;
```

Comment:
Consider making the port number configurable through environment variables. This will allow the application to be more flexible in different deployment environments. [important]